app/metrics.yaml

```diff
@@ -1856,3 +1856,16 @@ installation:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+
+perf_measurements:
+  gecko_runtime_create:
+    type: string
```

Comment:
Using a string metric type it's almost always a bad idea if the datum is not really a string. Coercing a number to a string makes it impossible for the SDK to validate data and to report errors. Moreover, it makes it impossible for the tooling (i.e. GLAM) to aggregate the data for you.

We tried to capture the spirit of this more in [our docs](https://mozilla.github.io/glean/book/user/adding-new-metrics.html).